filter-10-header.conf

```diff
@@ -0,0 +1,13 @@
+filter {
+   mutate {
+    add_field => {
+      "[bdpa][pipeline]" => "kubernetes"
+    }
+  }
+    grok {
+      match => ["[log][file][path]","/var/log/pods/%{DATA:kub_namespace}_%{DATA:kub_pod_name}_%{DATA:kub_pod_id}/%{DATA:kub_pod_container_name}/%{NOTSPACE:kub_contaner_logfile}"]
```

Comment:
Are we sure this is valid across Kubernetes distros? Might be that depending on how you would run Kubernetes this might path might change

@mocdaniel what do you think?

Reply:
Good catch! We built the pipeline on one single setup and tested it there. @mocdaniel should know which I'm talking about. ;-) The more universal it is, the better.

Reply:
To my knowledge, the default logs directory depends on the CRI, not the Kubernetes flavour/distro. Therefore, we could

a) decide that assumptions based on the usage of `containerd` are a sane default and **document this** default somewhere.
b) find out where e.g. `docker` and `cri-o` save their containers' logs to and add a parameter to set the used CRI.

Reply:
Since this is plain Logstash code, we can't use any variables to set values. All we have is `if` or RegExp.

Reply:
So there's no possibility to adapt the rules depending on which node is being scraped? This would suffice.

Reply:
No, I saw the comment, but then I tried to find a solution and *then* I forgot about it. 🙈 

* So, what we can do: Use hardcoded paths
* Use paths with regular expressions/shell globs
* Use different hardoced or RegExp paths like `if`, `else` and `else if`
* Basically nothing else

Documenting the default sounds like a plan. If you have alternatives for other paths, we can incorporate them.

Reply:
I think @Saeid-Abadi went ahead and merged a connected PR already and documented the default behaviour.